pkg/alert/alertconfigmap.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright (C) 2018 Synopsys, Inc.
```

Comment:
2019?

Reply:
Done!

Reply:
being nitty, but probably need to do for all new files :-)